data/sql/updates/pending_db_world/rev_1629131487005196300.sql

```diff
@@ -0,0 +1,57 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629131487005196300');
+
+-- Replace LANG_PINFO_MUTED
+UPDATE `acore_string` SET `content_default` = '│ Muted: (Reason: %s, Time: %s, Left %s, By: %s)',
+`locale_koKR` = NULL,
+`locale_frFR` = NULL,
+`locale_deDE` = NULL,
+`locale_zhCN` = NUll,
+`locale_zhTW` = NULL,
+`locale_esES` = NULL,
+`locale_esMX` = NULL,
+`locale_ruRU` = NULL
+WHERE `entry` = 550;
+
+-- LANG_COMMAND_DISABLE_CHAT_DELAYED
+UPDATE `acore_string` SET `content_default` = '%s has disabled %s\'s chat for %s, effective at the player\'s next login. Reason: %s.',
```

Comment:
```suggestion
UPDATE `acore_string` SET `content_default` = '%s has muted %s\'s chat for %s, effective at the player\'s next login. Reason: %s.',
```

Just for consistency.